metrics/metrics.go

```diff
@@ -19,20 +20,20 @@ type Metrics struct {
 }
 
 // NewMetrics creates an instance of metrics
-func NewMetrics(meter metric.Meter) *Metrics {
+func NewMetrics(meter metric.Meter, env, relayerID string) *Metrics {
 	totalRelayerCount := new(int64)
 	availableRelayerCount := new(int64)
 	return &Metrics{
 		ChainbridgeMetrics: *opentelemetry.NewChainbridgeMetrics(meter),
 		TotalRelayers: metric.Must(meter).NewInt64GaugeObserver(
-			"sygma.TotalRelayers",
+			fmt.Sprintf("sygma.{%s}.Relayer-{%s}.TotalRelayers", env, relayerID),
```

Comment:
Would it make more sense to have the id and env as context fields on the metric and not change metric name?
Also, do you need to add that to somemetrics from core?

Reply:
This sounds reasonable with context fields, although we are now not sure how to pass a meaningful context right now and just trying to make it work and build some alarm system with the OTLP agent and DataDog. 
About Core metrics, I am not sure we need the type of metrics served there, frankly speaking, those were pretty random. I do not want to use them for now

Reply:
We added meaningful stuff there in the mean time which we discussed we should have.